### ISSUE TITLE
Establish a11y office hours to help manage collaboration requests

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#63458]
## About your team

- Team name: Accessibility community of practice
- OCTO-DE product owner: @artsymartha68 
- Slack channel: #accessibility-help 
 
### Describe what you need in a few sentences:

I think it's time to establish weekly a11y office hours. A recent request to collaborate in an ad hoc meeting came through and a handful of a11y specialists were invited. While we appreciate the notion of reaching out to collaborate on an idea without the "overhead" of an official CAIA ticket, this is interruptive to folks who will be joining with no background or context. 

If we had a weekly meeting where random thoughts or concepts could be presented and discussed, this might alleviate the need for a random, ad hoc meeting with no agenda other than to "duke it out". A potential outcome of that discussion could be a CAIA ticket or a "back to the drawing board" if the concept is clearly unworkable.

@joshkimux has offered to host the first few with CAIA team members shadowing since so many folks are new to all the OCTO projects and people. 

I don't think all CAIA members need to attend all the office hours. Something we can have a sign-up for?

### Which are you doing?
Process improvement idea

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [x] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
### What's your team’s next step and its timing?

- **Our next step is** ....
- **Provide date if available**:  Can we start next week? 

#### Timing for your next step: 
  - [x] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

**Comment by joshkimux** [comment]
This is awesome, thanks Martha! I'd recommend an OCTO employee owning the calendar invite, just in case any turnover occurs into the future from contract shifts (so we can keep the same invite). Once we find a time, would you or another OCTO employee be willing to schedule the invite for it?

**Comment by sara-amanda** [comment]
## CAIA Update: Office Hours
Documenting information inventory I found on our sites, relating to accessibility help for @eli-oat. 

### Contact Us
This content needs updated. Notified @laurwill -[ it resides on the Platform website](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/contact-us). 

> Sitewide CAIA
> Project manager: Randi Hecht
> Information Architect (IA): TBD
> Managing editor: Laura Willwerth

### Office Hours
#### Calendly
This is [the tool that Platform uses](https://calendly.com/platform-office-hours) for reference.

#### Platform Website
Prepare for an accessibility staging review page [on the Platform site](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review).

> We recommend you `work with an accessibility specialist` to validate and resolve feedback received at your staging review. **If you do not have an accessibility specialist available to work with your team, please consider:**

> - Posting a question in the [#accessibility-help](https://dsva.slack.com/archives/C8E985R32) channel.

> - Attending [office hours](https://depo-platform-documentation.scrollhelp.site/support/office-hours) for either accessibility or for the Collaboration Cycle.

Topic | Date & Time | Meeting Info. | RSVP
-- | -- | -- | --
Accessibility | To be determined. For accessibility help, please attend Collaboration Cycle office hours and indicate that you want to speak with the accessibility specialist. | TBD | TBD

*Table found on this page: https://depo-platform-documentation.scrollhelp.site/support/office-hours

#### Intake Ticket
- The [intake ticket for CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request) found on Platform's CAIA page.
- It does not include any mention of office hours or the Slack channel. Any updates we make to office hours, might also be linked here?

cc: @coforma-terry

**Comment by coforma-terry** [comment]
Every Tuesday at 4PM ET

**Comment by sara-amanda** [comment]
## CAIA Update
**Reminders have been added to the following spaces.** 
- Timing for notifications per @coforma-terry, along with Zoom link in the reminder.
- @artsymartha68 ok'd adding to the `accessibility-help` channel via Slack. 
- **The initial Office Hours session begins:** 9/19/2023 at 4 p.m. EST
- cc: Leads: @kristinoletmuskat @laurwill 

### Slack Reminders Currently 
- [x] DSVA `accessibility-help` (the day before)
    - **Reminder**: TOMORROW: [CAIA](https://dsva.slack.com/archives/C01K37HRUAH) will hold office hours every Tuesday at 4 p.m. EST. :tada: Join our team members from Content, Accessibility and Information Architecture via [Zoom](https://coformaco.zoom.us/j/83992174780?pwd=Tk1jZ0o3MWxLcHFMWTM1S3ZrcytqUT09). If you have a question, please [use the thread inside the CAIA](https://dsva.slack.com/archives/C01K37HRUAH) channel, instead of the Zoom chat, to prevent screen reader interference.
- [x] DSVA `sitewide-content-accessibility-ia channel` (the day before)
    - **Reminder**: :announcement: TOMORROW: [CAIA](https://dsva.slack.com/archives/C01K37HRUAH) will hold office hours every Tuesday at 4 p.m. EST. :tada: Join our team members from Content, Accessibility and Information Architecture via [Zoom](https://coformaco.zoom.us/j/83992174780?pwd=Tk1jZ0o3MWxLcHFMWTM1S3ZrcytqUT09). If you have a question, please use this :thread: instead of the Zoom chat, to prevent screen reader interference.
- [x] Non-DSVA `proj-va-caia-core-team` (the day before)
    - **Reminder**: :speech_balloon: This message is being sent in multiple DSVA Channels today: :tada: [CAIA](https://dsva.slack.com/archives/C01K37HRUAH) will hold office hours every Tuesday at 4 p.m. EST. :tada: Join our team members from Content, Accessibility and Information Architecture via [Zoom](https://coformaco.zoom.us/j/83992174780?pwd=Tk1jZ0o3MWxLcHFMWTM1S3ZrcytqUT09). If you have a question, please [use the thread inside the CAIA](https://dsva.slack.com/archives/C01K37HRUAH) channel, instead of the Zoom chat, to prevent screen reader interference.
